### PR TITLE
Update install doc

### PR DESCRIPTION
**Pull request by nidup**

### CODE REVIEW

File: install_pim/installation_archive.rst.inc [hunk 1]
```diff
@@ -5,29 +5,53 @@ Extracting the archive
...
+
+    $ ../composer.phar install --optimize-autoloader --prefer-dist memory_limit=3G
+    $ yarn install
+
+Frontend,
+
+.. code-block:: bash
+    :linenos:
+
     $ php app/console cache:clear --env=prod
```
Comment (TheOnlyJoey) [inline install_pim/installation_archive.rst.inc]: Still references app/console, should be bin/console

**Comment by nidup** [comment]
@TheOnlyJoey exact! i changed it, still a lot of small things to do :wink:

**Comment by nidup** [comment]
more to come to install node & yarn :)

File: install_pim/installation_archive.rst.inc [hunk 2]
```diff
@@ -5,29 +5,53 @@ Extracting the archive
...
-    $ ../composer.phar install --optimize-autoloader --prefer-dist  # optional for community edition
-    $ php app/console cache:clear --env=prod
-    $ php app/console pim:install --env=prod
+
+Dependencies,
+
+.. code-block:: bash
+    :linenos:
+
+    $ ../composer.phar install --optimize-autoloader --prefer-dist memory_limit=3G
```
Comment (mickaelandrieu) [inline install_pim/installation_archive.rst.inc]:
```diff
- composer.phar install --optimize-autoloader --prefer-dist memory_limit=3G
+ php -d -d memory_limit=3G composer.phar install --optimize-autoloader --prefer-dist
```

Comment (mickaelandrieu) [reply install_pim/installation_archive.rst.inc]: nooo :'( too fast :/

Comment (nidup) [reply install_pim/installation_archive.rst.inc]: I changed it in upcoming PR https://github.com/akeneo/pim-docs/pull/673

Thank you mate :wink:

**Pull request merged**